test/cloudformation/iam_cloudformation.yaml

```diff
@@ -86,6 +86,7 @@ Resources:
             Resource:
               - !Sub "arn:${AWS::Partition}:iam::${AWS::AccountId}:instance-profile/KarpenterNodeInstanceProfile-*"
               - !Sub "arn:${AWS::Partition}:iam::${AWS::AccountId}:role/KarpenterNodeRole-*"
+              - !Sub "arn:${AWS::Partition}:iam::${AWS::AccountId}:role/GithubActionsRole*"
```

Comment:
Does this work? The instance profile name that you are using is different than this.

Reply:
We are just getting denied on profile creation, the role needs to authorized